docs/collecting-data/collecting-from-own-applications/javascript-trackers/node-js-tracker/node-js-tracker-v3/initialization/index.md

```diff
@@ -31,7 +31,7 @@ A simple set up of this emitter might look like:
 ```
 const e = gotEmitter(
   'collector.mydomain.net', // Collector endpoint
-  snowplow.HttpProtocol.HTTPS, // Optionally specify a method - http is the default
+  snowplow.HttpProtocol.HTTPS, // Optionally specify a method - https is the default
```

Comment:
Is there actually a default? It seems like it is a required property for the `gotEmitter` call and I don't see a place where the default is assigned, but I might just be confused.